### PR TITLE
AOA and SSA message definitions

### PR DESCRIPTION
**Pull request by EShamaev**
Message ID's are in 0-255 range for version 1 of Mavlink
This messages are for https://github.com/ArduPilot/ardupilot/pull/5933

### CODE REVIEW

**Comment by magicrub** [comment]
Mind moving this to common?

**Comment by EShamaev** [comment]
@magicrub Done

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -3587,6 +3587,11 @@
       <field type="float" name="size_y">Size in radians of target along y-axis</field>
     </message>
     <!-- MESSAGE IDs 180 - 229: Space for custom messages in individual projectname_messages.xml files -->
+    <message id="227" name="AOA_SSA">
+      <description>Angle of Attack and Side Slip Angle</description>
+      <field name="AOA" type="float">Angle of Attack (degrees)</field>
```
Comment (tridge) [inline message_definitions/v1.0/common.xml]: I think we should add a 32 bit time_boot_ms, to make it easier to graph
most data mavlink msgs have a timestamp

Comment (EShamaev) [inline message_definitions/v1.0/common.xml]: Timestamp added. Also updated the Mission Planner with new definition.

**Comment by WickedShell** [comment]
Thats the wrong ID. That's a project specific ID, it was fine when it was in ardupilotmega.xml, but can't go in common with that ID.

**Comment by magicrub** [comment]
any reason this can't go into the higher MAVLink v2.0 address space?

**Comment by EShamaev** [comment]
Somehow it does not get processed by MP.
I will find out and move to higher address space.

**Comment by tridge** [comment]
applied, thanks!

**Pull request closed without merging**